docs/user/alerting/alerting-troubleshooting.asciidoc

```diff
@@ -69,3 +69,168 @@ Configuration options are available to specialize connections to TLS servers,
 including ignoring server certificate validation, and providing certificate
 authority data to verify servers using custom certificates.  For more details, 
 see <<action-settings,Action settings>>.
+
+[float]
+[[rules-long-execution-time]]
+=== Identify long running rules
```

Comment:
We'll need to add a paragraph / section here talking about the fact that this index is a Kibana system index.  By default, no normal users should be able to read it.  Superuser can.  And presumably there is a way to allow users access to this, but I'm not quite sure what the best approach is.  Probably need to update their user to add a read priv for this index - or actually all the matching indices, and it should probably cover the versions as well, so `.kibana-event-log-*` or whatever.

Reply:
Added https://github.com/elastic/kibana/pull/98773/commits/5a555f400ba87dce1759e4c298a46b26b989b0f2